src/components/modal/ModalBody.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import { ModalBody as ChakraModalBody, ModalBodyProps } from '@chakra-ui/react';
+
+const ModalBody: FC<PropsWithChildren<ModalBodyProps>> = (props) => {
+  const { children, ...modalBodyProps } = props;
+
+  return <ChakraModalBody {...modalBodyProps}>{children}</ChakraModalBody>;
```

Comment:
What we get with this wrapping of Chakra Modal. Why don't we just export Ch ModalBody component?